app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -75,6 +75,8 @@
 import Vue from 'vue';
 import { mapActions, mapGetters } from 'vuex';
 import { Form } from 'vue-formio';
+import templateExtensions from '@/plugins/templateExtensions';
+Form.use(templateExtensions);
```

Comment:
These lines should be inserted after the second block of imports because templateExtensions is something we wrote, not an external library import.